test/test.config.js

```diff
@@ -20,3 +21,24 @@ config.mongodb.password = 'password';
 config.mongodb.adminPrompt = true;
 config.mongodb.dropCollections.onInit = true;
 config.mongodb.dropCollections.collections = [];
+
+constants.SECURITY_CONTEXT_V1_URL = 'https://w3id.org/security/v1';
+constants.CONTEXTS[constants.SECURITY_CONTEXT_V1_URL] = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, 'contexts/security-v1.jsonld'),
+    {encoding: 'utf8'}));
+constants.SECURITY_CONTEXT_V2_URL = 'https://w3id.org/security/v2';
+constants.CONTEXTS[constants.SECURITY_CONTEXT_V2_URL] = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, 'contexts/security-v2.jsonld'),
+    {encoding: 'utf8'}));
+constants.IDENTITY_CONTEXT_V1_URL = 'https://w3id.org/identity/v1';
+constants.CONTEXTS[constants.IDENTITY_CONTEXT_V1_URL] = JSON.parse(
+  fs.readFileSync(
+    path.join(__dirname, 'contexts/identity-v1.jsonld'),
+    {encoding: 'utf8'}));
+constants.SCHEMA_URL = 'https://schema.org/';
```

Comment:
Is this needed?  Would be good to avoid adding this and the entire schema context file just for testing.

Reply:
I guess this is pulled in indirectly through the identity context?  Maybe we could just pull out the few properties we need into the local schema context?

Reply:
I don't think we should be pulling in 130K+ files just for testing like that.  Upon further consideration, I don't think it's even used anywhere?  We ref the schema.org ids, but does any test actually load the context?

Reply:
Removed.